tests/e2e/test_rosacli_cluster.go

```diff
@@ -854,6 +855,126 @@ var _ = Describe("Classic cluster creation validation",
 				Expect(err).NotTo(BeNil())
 				Expect(errorOutput.String()).To(ContainSubstring("etcd encryption cannot be disabled on clusters with FIPS mode"))
 			})
+
+		It("Create rosa cluster with additional security groups will validate well via rosacli - [id:68971]",
+			labels.Medium, labels.Runtime.Day1Negative,
+			func() {
+				var (
+					ocpVersionBelow4_14 = "4.13.12"
+					ocpVersion4_14      = "4.14.0"
+					index               int
+					flagName            string
+					securityGroups      = map[string]string{
+						"--additional-infra-security-group-ids":         "sg-aisgi",
+						"--additional-control-plane-security-group-ids": "sg-acpsgi",
+						"--additional-compute-security-group-ids":       "sg-acsgi",
+					}
+
+					invalidSecurityGroups = map[string]string{
+						"--additional-infra-security-group-ids":         "invalid",
+						"--additional-control-plane-security-group-ids": "invalid",
+						"--additional-compute-security-group-ids":       "invalid",
+					}
+				)
+				profile := profilehandler.LoadProfileYamlFileByENV()
+
+				By("Prepare creation command")
+				var command string
+				var rosalCommand config.Command
+				flags, err := profilehandler.GenerateClusterCreateFlags(profile, rosaClient)
+				Expect(err).To(BeNil())
+
+				command = "rosa create cluster --cluster-name " + profile.ClusterConfig.Name + " " + strings.Join(flags, " ")
+				rosalCommand = config.GenerateCommand(command)
+				for additionalSecurityGroupFlag := range securityGroups {
+					rosalCommand.DeleteFlag(additionalSecurityGroupFlag, true)
+				}
+
+				if profile.ClusterConfig.HCP {
+					for additionalSecurityGroupFlag, value := range securityGroups {
+						rosalCommand.AddFlags("--dry-run",
+							additionalSecurityGroupFlag, value,
+							"--version", "4.15.11",
+							"--hosted-cp", "-y")
+						output, err := rosaClient.Runner.RunCMD(strings.Split(rosalCommand.GetFullCommand(), " "))
+						Expect(err).To(HaveOccurred())
+						index = strings.Index(additionalSecurityGroupFlag, "a")
+						flagName = additionalSecurityGroupFlag[index:]
+						Expect(output.String()).To(ContainSubstring("Using '301721915996' as billing account"))
```

Comment:
nit: could we make this billing account a constant?

Reply:
This is sensitive info. Don't show billing account in code.
```suggestion
						Expect(output.String()).To(MatchRegex(MatchRegexp(`Using '[0-9]+' as billing account`)))
```

Reply:
This is not what we want to verify. Remove line 903 - 905